Script throws exception when appointments available

**Describe the issue**
When a Vermittlungscode is present and the script searches for appointments, as soon as appointments get available, the script throws an error (see below). It seems the `send_alert()` method is used wrongly.

**Versions**
macOS 11.3, impf-botpy 0.41

**Log files**
```
2021-05-24 11:14:51,964 - [WARNING] <impf.browser.browser.py:253>  <PLZ>: Available appointments! Waiting for user input
2021-05-24 11:14:51,976 - [ERROR] <impf.browser.decorators.py:81>  <PLZ>: An unexpected exception occurred in <control_appointment>
Traceback (most recent call last):
  File "/Users/senpo/src/impf-botpy/impf/decorators.py", line 63, in func
    return f(self, *args, **kwargs)
  File "/Users/senpo/src/impf-botpy/impf/browser.py", line 449, in control_appointment
    self.alert_appointment()
  File "/Users/senpo/src/impf-botpy/impf/browser.py", line 257, in alert_appointment
    send_alert()
TypeError: send_alert() missing 1 required positional argument: 'message'
2021-05-24 11:15:01,987 - [INFO] <__main__.main.py:95>  Waiting until 11:20:01 before checking the next location
```

Thanks.

--- a/impf/browser.py
+++ b/impf/browser.py
@@ -254,7 +254,7 @@ class Browser:
         alert = settings.ALERT_AVAILABLE\
             .replace('{{ LOCATION }}', self.location_full)\
             .replace('{{ LINK }}', self.driver.current_url)
-        send_alert()
+        send_alert(alert)
         self.keep_browser = True
 
         if not settings.BOOK_REMOTELY:

